Source/Widgets/InfoBox/InfoBox.js

```diff
@@ -68,21 +69,56 @@ click: function () { closeClicked.raiseEvent(this); }');
         closeElement.innerHTML = '&times;';
         infoElement.appendChild(closeElement);
 
-        var infoBodyElement = document.createElement('div');
-        infoBodyElement.className = 'cesium-infoBox-body';
-        infoElement.appendChild(infoBodyElement);
-
-        var descriptionElement = document.createElement('div');
-        descriptionElement.className = 'cesium-infoBox-description';
-        descriptionElement.setAttribute('data-bind', '\
-html: descriptionSanitizedHtml,\
-style : { maxHeight : maxHeightOffset(40) }');
-        infoBodyElement.appendChild(descriptionElement);
+        var frame = document.createElement('iframe');
+        frame.className = 'cesium-infoBox-iframe';
+        frame.setAttribute('data-bind', 'style : { maxHeight : maxHeightOffset(40) }');
+        frame.setAttribute('sandbox', 'allow-same-origin allow-popups allow-pointer-lock allow-forms'); // allow-scripts allow-top-navigation
```

Comment:
Given that `allow-scripts` is off by default, I don't think there's any reason to have `allow-same-origin` or `allow-pointer-lock`, unless you can think of a reason.  `allow-popups` allows `target="_blank"` which seems reasonable.  `allow-forms` is somewhat tough to think of a use for, given the lack of scripts, but it seems ok.  Perhaps you could have a plain HTML form which submits to the server which could trigger a server-sent event to load new CZML or something.